app/src/app/components/view-profile/view-profile.component.spec.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright © 2005 - 2021 Alfresco Software, Ltd. All rights reserved.
+ *
+ * License rights for this program may be obtained from Alfresco Software, Ltd.
+ * pursuant to a written agreement and any use of this program without such an
+ * agreement is prohibited.
+ */
+import { TestBed, ComponentFixture } from '@angular/core/testing';
+import { AppConfigModule } from '@alfresco/adf-core';
+import { ViewProfileComponent } from './view-profile.component';
+import { AppTestingModule } from '../../testing/app-testing.module';
+
+describe('ViewProfileComponent', () => {
```

Comment:
should contain tests for all the features you have added

Reply:
Added the test cases for input field validations.